File: PRIMERPARCIAL_SIS104/app/src/main/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {


    Button buttEjer1, buttEjer2, buttEjer3;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        buttEjer1 = findViewById(R.id.buttonEjer1);
        buttEjer2 = findViewById(R.id.buttonEjer2);
        buttEjer3 = findViewById(R.id.buttonEjer3);

        buttEjer1.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent =  new Intent(MainActivity.this, Ejercicio1Activity.class);
                startActivity(intent);
            }
        });

        buttEjer2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent =  new Intent(MainActivity.this, Ejercicio2Activity.class);
                startActivity(intent);
            }
        });

        buttEjer3.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent =  new Intent(MainActivity.this, Ejercicio3Activity.class);
                startActivity(intent);
            }
        });



    }
}

File: SEGUNDOPARCIAL_SIS104/app/src/main/java/com/example/segundoparcial_sis104/MainActivity.kt
package com.example.segundoparcial_sis104

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val buttMostrarPuntos = findViewById<Button>(R.id.buttonMostrarPuntos)
        val buttSalir = findViewById<Button>(R.id.buttonSalir)

        buttMostrarPuntos.setOnClickListener {
            val intent = Intent(this@MainActivity, SqlActivity::class.java)
            startActivity(intent)
        }

        buttSalir.setOnClickListener {
            finish()
        }

    }
}